### ISSUE TITLE
Dropdown toggle aria-label

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-react#568]
_NOTE: This issue might already be addressed in PR https://github.com/patternfly/patternfly-react/pull/506, but the preview doesn't seem to be working so I can't review to confirm. Just capturing this here so that it doesn't get lost._

In the Dropdown component, the `aria-label` attribute should only be present in the instances where a text label is not visible in the button that toggles the menu. For example, the kebab example should have something like `aria-label="Actions"` but the other examples that use "Dropdown" as the button label should not have this attribute present.
